hq/app/services/CacheService.scala

```diff
@@ -0,0 +1,49 @@
+package services
+
+import aws.ec2.EC2
+import com.gu.Box
+import config.Config
+import model.{AwsAccount, SGInUse, SGOpenPortsDetail}
+import play.api.inject.ApplicationLifecycle
+import play.api.{Configuration, Environment, Mode}
+import rx.lang.scala.Observable
+import utils.attempt.{FailedAttempt, Failure}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+
+
+class CacheService(config: Configuration, lifecycle: ApplicationLifecycle, environment: Environment)(implicit ec: ExecutionContext) {
+  private val sgsBox: Box[Map[AwsAccount, Either[FailedAttempt, List[(SGOpenPortsDetail, Set[SGInUse])]]]] = Box(Map.empty)
+  private val accounts = Config.getAwsAccounts(config)
+
+  def getAllSgs(): Map[AwsAccount, Either[FailedAttempt, List[(SGOpenPortsDetail, Set[SGInUse])]]] = sgsBox.get()
```

Comment:
We did talk about initialising this data. Happy to defer this to another PR if you like, but I feel like it might be worth doing this to distinguish between accounts that aren't found and accounts that haven't yet been updated.

Reply:
Yep, sounds sensible! 👍 
I would like to do a followup PR for setting starting values in each box